src/test/java/com/github/romankh3/image/comparison/ImageComparisonUnitTest.java

```diff
@@ -464,7 +465,9 @@ public void shouldProperlyWorkGettersAndSetters() {
                 .setThreshold(400)
                 .setDifferenceRectangleFilling(true, 35.1)
                 .setExcludedRectangleFilling(true, 45.1)
-                .setAllowingPercentOfDifferentPixels(48.15);
+                .setAllowingPercentOfDifferentPixels(48.15)
+                .setDifferenceRectangleColor(Color.RED)
```

Comment:
It's good to set a non-default value here. The same for line 469.
Please, update it.

Reply:
Thank you for your comment.
Could you please let me know that you mean only set a non-default value for test class or for main modification as well?

Reply:
Only for test case to be ensure, that setter works as well as we expected. 

Default values for production code should be the same as were before.